spec/unit/pdk/cli/env_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'spec_helper'
+require 'pdk/cli'
+
+describe 'Running `pdk env`' do
+  let(:command_args) { ['env'] }
+  let(:command_result) { { exit_code: 0 } }
+
+  context 'when it calls env successfully' do
+    after(:each) do
+      expect {
+        PDK::CLI.run(command_args)
+      }.to exit_zero
+    end
+
+    before(:each) do
+      allow(PDK::Util::RubyVersion).to receive(:gem_home).and_return('/opt/puppetlabs/pdk/share/cache/ruby/2.4.0')
+      allow(PDK::Util::RubyVersion).to receive(:gem_path).and_return('/opt/puppetlabs/pdk/private/ruby/2.4.3/lib')
+      allow(PDK::Util::RubyVersion).to receive(:bin_path).and_return('/opt/puppetlabs/pdk/private/ruby/2.4.3/bin')
+      allow(PDK::Util::RubyVersion).to receive(:gem_paths_raw).and_return(['/opt/puppetlabs/pdk/private/ruby/2.4.3/lib'])
+      allow(PDK::Util::Env).to receive(:[]).and_call_original
+      allow(PDK::Util::Env).to receive(:[]).with('PATH').and_return('/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin')
```

Comment:
I have a concern with using hardcoded nix/macOS type paths here in the spec tests. To me, this should fail becasue those paths do not exist on Windows, but Appveyor is green, so this means either we are not running these tests in appevyor or these tests aren't testing what we think they are?

I realize this is an experimental command, but I don't think this should merge unless we confirm this works on all three platforms. If precedent says PDK doesn't test on windows, then I can add that as a action item for me going forward and can merge this ticket as is

Reply:
Since this is just mocking the return values and not asserting anything about them, I suppose they could return just about anything, e.g. `'fake return path'` or `C:\Windows\Path`. If they aren't mocked, they would presumably return values from the host running the tests, which I thought wasn't totally ideal for a unit test.

To be honest though, I'm not sure of the best way to test this on windows. @scotje do you have any context on PDK and windows testing?

Reply:
@scotje commented that these are unit level tests, so mocking should be OK. I made a mistake and misread these as acceptance level tests. I'll ticket future work to add acceptance level tests and merge as is